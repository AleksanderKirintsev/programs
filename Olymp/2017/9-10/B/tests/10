10
44 90
4 68
89 89
52 93
44 56
86 99
100 100
95 99
89 89
62 76
