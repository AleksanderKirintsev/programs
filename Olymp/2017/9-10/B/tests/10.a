69 90
4 68
-1 -1
91 93
-1 -1
94 99
100 100
-1 -1
-1 -1
-1 -1
