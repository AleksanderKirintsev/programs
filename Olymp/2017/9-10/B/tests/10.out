-1 -1
4 68
-1 -1
69 93
-1 -1
94 99
100 100
95 99
-1 -1
69 76
